module/VuFindSearch/src/VuFindSearch/Backend/AbstractBackend.php

```diff
@@ -116,6 +119,27 @@ abstract public function getRecordCollectionFactory();
     protected function injectSourceIdentifier(RecordCollectionInterface $response)
     {
         $response->setSourceIdentifiers($this->identifier);
+        $response->setResultSetIdentifier($this->generateUuid());
```

Comment:
I think this approach is fine for now, but maybe it's worth putting a TODO comment here to switch to using the Doctrine UUID generator once it becomes available to us in the future. Then we could potentially address that as part of (or sometime after the merge of) #2233.